README.md

```diff
@@ -1,6 +1,7 @@
 # blue-green-deployment
 
 # Local Kubernetes Deployment
+# Manual deploy

```

Comment:
Step By Step Manual Deployment would be more appropriate. and below could be something like Automated Script Deployment instead of deploy all. what do you think?

Reply:
done all changes